static/app/utils/sessions.tsx

```diff
@@ -198,7 +198,7 @@ export function getCountSeries(
 }
 
 export function initSessionsChart(theme: Theme) {
-  const colors = theme.charts.getColorPalette(14);
+  const colors = theme.charts.getColorPalette(14) ?? [];
```

Comment:
feels like it should always return an array

Reply:
I think so, but only if we make it throw on out of bounds access :D